src/config/dynamic-theme-fixes.config

```diff
@@ -546,6 +546,9 @@ INVERT
 .ms-Grid-col.ms-sm8.ms-lg4
 .ms-Grid-col.ms-sm4.ms-hiddenMdDown.pullout-textwrap
 .ms-Grid-col.ms-sm2 img
+.pullout-icon:not(.ms-Icon--SkypeCircleCheck)
```

Comment:
You just messed a little bit with copy paste...
```
.pullout-link-text
.pullout-icon:not(.ms-Icon--SkypeCircleCheck)
.pullout-textwrap span
```
That's what we wanted to have in the end. 😉

Reply:
I am sorry for that, 😅